### DISCUSSION TITLE
Modell compressed by draco crashes on mobile

### DISCUSSION THREAD

**Discussion by Aztec91** [google/model-viewer#2309] [Q&A]
Hey Guys, its me again.

I had this hughe model, it was about 180MB and I exportet it in Blender with Draco to 12MB all in All. The Loading time on Desktop is really 90% faster now, but my mobile phone still cant handle the 12MB... sometimes it displays the model for 1 sec and then crashes.

In other cases I had Modells about 60MB that my phone could open (but I didnt have to compress it first).

Here is my working example:

https://commacross.de/3DAR/motor.html

I know this models has like 3 Million Faces / Vertices etc... but the phone should be able to handle the size right? With a 1 Mil. Faces / Vertices my phone had no problem (60MB uncompressed)

Regards
Aztec

**Comment by elalish** [comment]
Well, it probably depends on the phone; what are you testing with? Considering the number of pixels you're drawing to on a phone, it's unlikely those 3 million faces are helping much with render quality. I'd recommend decimation, as that will help performance as well as crashes. You may also want to check on your textures; Basis compression may help there as it reduces the GPU RAM footprint considerably.

**Reply by Aztec91** [reply]
Thank you for the answer. So is Material also considered a Texturen? Also I wonder, how for example does sketchfab Display such hughe Models? With Premium Account you can upload up to 500MB if I read right. Do they have a special thechnique? I teste on a xiaomi redmi Note 9T. Not the best Phone but Not too Bad either.

**Reply by elalish** [reply]
A material can have several textures, or even none. However, for a given property (say baseColor) there can only be a maximum of one texture per material. As for sketchfab, I'm not sure, but I wouldn't be surprised if they simply perform the decimation for you to create a more web-optimized asset.
